Crash when loading heads with broken textures

Friend's creative server has heads that point to now non-existent URLs, and Ears trying to process them causes a crash since they aren't an image format anymore and instead 404 pages.

<details>
<summary>Crash log</summary>
<pre>
[10:50:16] [main/WARN]: Texture Loading Finished
[10:50:17] [main/WARN]: Error while loading the skin texture
java.io.IOException: Could not load image: Image not of any known type, or corrupt
	at net.minecraft.class_1011.method_4303(class_1011.java:125) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1011.method_4310(class_1011.java:99) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1011.method_4309(class_1011.java:90) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1046.method_22795(class_1046.java:141) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1046.method_4625(class_1046.java:91) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1060.method_24303(class_1060.java:96) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1060.method_4616(class_1060.java:68) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1071.method_4651(class_1071.java:80) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1071.method_4656(class_1071.java:60) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3578(class_836.java:103) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3581(class_836.java:86) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3577(class_836.java:65) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3569(class_836.java:29) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_23079(class_824.java:107) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_23081(class_824.java:96) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_3554(class_824.java:128) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_3555(class_824.java:96) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_761.method_22710(class_761.java:1260) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_757.method_3188(class_757.java:727) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:546) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1048) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:681) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:215) [intermediary-minecraft-1.16.5-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
[10:50:18] [main/FATAL]: Reported exception thrown!
net.minecraft.class_148: Registering texture
	at net.minecraft.class_824.method_3554(class_824.java:135) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_3555(class_824.java:96) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_761.method_22710(class_761.java:1260) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_757.method_3188(class_757.java:727) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:546) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1048) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:681) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:215) [intermediary-minecraft-1.16.5-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.class_1046.handler$bke000$loadTexture(class_1046.java:536) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1046.method_22795(class_1046.java:148) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1046.method_4625(class_1046.java:91) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1060.method_24303(class_1060.java:96) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1060.method_4616(class_1060.java:68) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1071.method_4651(class_1071.java:80) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_1071.method_4656(class_1071.java:60) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3578(class_836.java:103) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3581(class_836.java:86) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3577(class_836.java:65) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_836.method_3569(class_836.java:29) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_23079(class_824.java:107) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_23081(class_824.java:96) ~[intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_824.method_3554(class_824.java:128) ~[intermediary-minecraft-1.16.5-client.jar:?]
	... 22 more
---- Minecraft Crash Report ----
// My bad.

Time: 4/19/21, 10:50 AM
Description: Registering texture

java.lang.NullPointerException: Registering texture
	at net.minecraft.class_1046.handler$bke000$loadTexture(class_1046.java:536)
	at net.minecraft.class_1046.method_22795(class_1046.java:148)
	at net.minecraft.class_1046.method_4625(class_1046.java:91)
	at net.minecraft.class_1060.method_24303(class_1060.java:96)
	at net.minecraft.class_1060.method_4616(class_1060.java:68)
	at net.minecraft.class_1071.method_4651(class_1071.java:80)
	at net.minecraft.class_1071.method_4656(class_1071.java:60)
	at net.minecraft.class_836.method_3578(class_836.java:103)
	at net.minecraft.class_836.method_3581(class_836.java:86)
	at net.minecraft.class_836.method_3577(class_836.java:65)
	at net.minecraft.class_836.method_3569(class_836.java:29)
	at net.minecraft.class_824.method_23079(class_824.java:107)
	at net.minecraft.class_824.method_23081(class_824.java:96)
	at net.minecraft.class_824.method_3554(class_824.java:128)
	at net.minecraft.class_824.method_3555(class_824.java:96)
	at net.minecraft.class_761.method_22710(class_761.java:1260)
	at net.minecraft.class_757.method_3188(class_757.java:727)
	at net.minecraft.class_757.method_3192(class_757.java:546)
	at net.minecraft.class_310.method_1523(class_310.java:1048)
	at net.minecraft.class_310.method_1514(class_310.java:681)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_1046.handler$bke000$loadTexture(class_1046.java:536)
	at net.minecraft.class_1046.method_22795(class_1046.java:148)
	at net.minecraft.class_1046.method_4625(class_1046.java:91)

-- Resource location being registered --
Details:
	Resource location: minecraft:skins/4d78a26e6d27fcfe02fce139bdbb407790b95dd6
	Texture object class: net.minecraft.class_1046
Stacktrace:
	at net.minecraft.class_1060.method_24303(class_1060.java:96)
	at net.minecraft.class_1060.method_4616(class_1060.java:68)
	at net.minecraft.class_1071.method_4651(class_1071.java:80)
	at net.minecraft.class_1071.method_4656(class_1071.java:60)
	at net.minecraft.class_836.method_3578(class_836.java:103)
	at net.minecraft.class_836.method_3581(class_836.java:86)
	at net.minecraft.class_836.method_3577(class_836.java:65)
	at net.minecraft.class_836.method_3569(class_836.java:29)
	at net.minecraft.class_824.method_23079(class_824.java:107)
	at net.minecraft.class_824.method_23081(class_824.java:96)

-- Block Entity Details --
Details:
	Name: minecraft:skull // net.minecraft.class_2631
	Block: Block{minecraft:player_head}[rotation=10]
	Block location: World: (57,66,7), Chunk: (at 9,4,7 in 3,0; contains blocks 48,0,0 to 63,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Block: Block{minecraft:player_head}[rotation=10]
	Block location: World: (57,66,7), Chunk: (at 9,4,7 in 3,0; contains blocks 48,0,0 to 63,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
Stacktrace:
	at net.minecraft.class_824.method_3554(class_824.java:128)
	at net.minecraft.class_824.method_3555(class_824.java:96)
	at net.minecraft.class_761.method_22710(class_761.java:1260)
	at net.minecraft.class_757.method_3188(class_757.java:727)

-- Affected level --
Details:
	All players: 1 total; [class_746['Cynosphere'/36, l='ClientLevel', x=48.07, y=77.99, z=65.60]]
	Chunk stats: Client Chunk Cache: 729, 490
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,5,0), Chunk: (at 0,0,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 424388569 game time, 11392050 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:449)
	at net.minecraft.class_310.method_1587(class_310.java:2239)
	at net.minecraft.class_310.method_1514(class_310.java:699)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 11.0.9.1, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), AdoptOpenJDK
	Memory: 3354272768 bytes (3198 MB) / 4294967296 bytes (4096 MB) up to 4294967296 bytes (4096 MB)
	CPUs: 16
	JVM Flags: 9 total; -XX:+UseG1GC -XX:+UnlockExperimentalVMOptions -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=25 -XX:G1HeapRegionSize=32M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms4096m -Xmx4096m
	Mixins in Stacktrace: 
		net.minecraft.class_1046:
			com.minelittlepony.common.mixin.MixinPlayerSkinTexture (kirin.mixin.json)
			com.unascribed.ears.mixin.MixinPlayerSkinTexture (ears.mixins.json)
		net.minecraft.class_1060:
			com.unascribed.fabrication.mixin.a_fixes.better_pause_freezing.MixinTextureManager (fabrication.mixins.json)
			net.fabricmc.fabric.mixin.resource.loader.client.KeyedResourceReloadListenerClientMixin (fabric-resource-loader-v0.mixins.json)
		net.minecraft.class_1071:
			me.pepperbell.anycapes.mixin.PlayerSkinProviderMixin (anycapes.mixins.json)
			com.minelittlepony.common.mixin.MixinPlayerSkinProvider (kirin.mixin.json)
		net.minecraft.class_824:
			fi.dy.masa.tweakeroo.mixin.MixinBlockEntityRenderDispatcher (mixins.tweakeroo.json)
			net.fabricmc.fabric.mixin.client.renderer.registry.MixinBlockEntityRenderDispatcher (fabric-renderer-registries-v1.mixins.json)
		net.minecraft.class_761:
			me.lambdaurora.lambdynlights.mixin.CommonWorldRendererMixin (lambdynlights.mixins.json)
			net.fabricmc.fabric.mixin.client.rendering.MixinWorldRenderer (fabric-rendering-v1.mixins.json)
			fi.dy.masa.malilib.mixin.MixinWorldRenderer (mixins.malilib.json)
			fi.dy.masa.tweakeroo.mixin.MixinWorldRenderer (mixins.tweakeroo.json)
			fi.dy.masa.minihud.mixin.IMixinWorldRenderer (mixins.minihud.json)
			fi.dy.masa.litematica.mixin.MixinWorldRenderer (mixins.litematica.json)
			me.lambdaurora.lambdacontrols.client.mixin.WorldRendererMixin (lambdacontrols.mixins.json)
			net.blancworks.figura.mixin.WorldRendererMixin (figura.mixins.json)
			net.fabricmc.fabric.mixin.resource.loader.client.KeyedResourceReloadListenerClientMixin (fabric-resource-loader-v0.mixins.json)
			me.shedaniel.cloth.mixin.client.events.MixinWorldRenderer (cloth-client-events-v0.mixins.json)
			net.torocraft.torohealth.mixin.WorldRendererMixin (torohealth.mixins.json)
		net.minecraft.class_757:
			pm.c7.utils.mixin.fullbright.MixinGameRenderer (blanket.mixins.json)
			io.github.jsnimda.inventoryprofiles.mixin.MixinGameRenderer (mixins.inventoryprofiles.json)
			me.shedaniel.slightguimodifications.mixin.MixinGameRenderer (slight-gui-modifications.mixins.json)
			me.lambdaurora.lambdacontrols.client.mixin.GameRendererMixin (lambdacontrols.mixins.json)
			fi.dy.masa.tweakeroo.mixin.MixinGameRenderer (mixins.tweakeroo.json)
			me.shedaniel.cloth.mixin.client.events.MixinGameRenderer (cloth-client-events-v0.mixins.json)
			win.baruna.blockmeter.mixin.GameRendererMixin (blockmeter.mixins.json)
			com.unascribed.fabrication.mixin.a_fixes.no_night_vision_flash.MixinGameRenderer (fabrication.mixins.json)
			io.github.joaoh1.okzoomer.client.mixin.GameRendererMixin (okzoomer.mixins.json)
			me.shedaniel.architectury.mixin.fabric.client.MixinGameRenderer (architectury.mixins.json)
			net.fabricmc.fabric.mixin.screen.GameRendererMixin (fabric-screen-api-v1.mixins.json)
		net.minecraft.class_310:
			me.shedaniel.advancementsenlarger.mixin.MixinMinecraftClient (advancements-enlarger.mixins.json)
			net.fabricmc.fabric.mixin.screen.MinecraftClientMixin (fabric-screen-api-v1.mixins.json)
			fi.dy.masa.tweakeroo.mixin.MixinMinecraftClient (mixins.tweakeroo.json)
			me.shedaniel.slightguimodifications.mixin.MixinMinecraft (slight-gui-modifications.mixins.json)
			net.fabricmc.fabric.mixin.networking.accessor.MinecraftClientAccessor (fabric-networking-api-v1.mixins.json)
			fi.dy.masa.litematica.mixin.MixinMinecraftClient (mixins.litematica.json)
			fi.dy.masa.minihud.mixin.MixinMinecraftClient (mixins.minihud.json)
			me.shedaniel.cloth.mixin.client.events.MixinMinecraftClient (cloth-client-events-v0.mixins.json)
			com.unascribed.fabrication.mixin.a_fixes.uncap_menu_fps.MixinMinecraftClient (fabrication.mixins.json)
			me.lambdaurora.lambdynlights.mixin.MinecraftClientMixin (lambdynlights.mixins.json)
			net.fabricmc.fabric.mixin.event.interaction.MixinMinecraftClient (fabric-events-interaction-v0.mixins.json)
			io.github.joaoh1.okzoomer.client.mixin.MinecraftClientMixin (okzoomer.mixins.json)
			net.fabricmc.fabric.mixin.command.client.MinecraftClientMixin (fabric-command-api-v1.mixins.json)
			me.lambdaurora.lambdacontrols.client.mixin.MinecraftClientMixin (lambdacontrols.mixins.json)
			com.blamejared.controlling.mixin.OpenGuiMixin (controlling.mixins.json)
			com.swordglowsblue.artifice.mixin.MixinMinecraftClient (artifice.mixins.json)
			squeek.quakemovement.mixin.KeyPressMixin (squake.mixins.json)
			me.shedaniel.architectury.mixin.fabric.client.MixinMinecraft (architectury.mixins.json)
			me.lambdaurora.spruceui.mixin.MinecraftClientMixin (spruceui.mixins.json)
			net.fabricmc.fabric.mixin.registry.sync.client.MixinMinecraftClient (fabric-registry-sync-v0.mixins.json)
			net.earthcomputer.multiconnect.mixin.bridge.MinecraftClientAccessor (multiconnect.mixins.json)
			de.siphalor.mousewheelie.client.mixin.MixinMinecraftClient (mousewheelie.mixins.json)
			squeek.appleskin.mixin.MinecraftClientMixin (appleskin.mixins.json)
			net.fabricmc.fabric.mixin.event.lifecycle.client.MinecraftClientMixin (fabric-lifecycle-events-v1.mixins.json)
			pm.c7.utils.mixin.MixinTitleBar (blanket.mixins.json)
			io.github.jsnimda.inventoryprofiles.mixin.MixinMinecraftClient (mixins.inventoryprofiles.json)
			com.rdvdev2.disablecustomworldsadvice.mixin.MixinMinecraftClient (disable_custom_worlds_advice.mixins.json)
			net.earthcomputer.multiconnect.mixin.connect.MixinMinecraftClient (multiconnect.mixins.json)
			net.earthcomputer.multiconnect.protocols.v1_11_2.mixin.MixinMinecraftClient (multiconnect.1_11_2.mixins.json)
			fi.dy.masa.malilib.mixin.MixinMinecraftClient (mixins.malilib.json)
		net.minecraft.client.main.Main:
			io.github.ultimateboomer.smoothboot.mixin.client.MainMixin (smoothboot.mixins.json)
	Fabric Mods: 
		advancementinfo: AdvancementInfo 1.16.5-fabric0.30.0-1.2
		advancements-enlarger: Advancements Enlarger 0.2.3
		amecsapi: Amecs API 1.1.3+mc1.16.5
		antighost: AntiGhost 1.16.5-fabric0.30.0-1.1.3
		anycapes: AnyCapes 1.0.2
		appleskin: AppleSkin 1.0.11
		architectury: Architectury 1.8.128
		artifice: Artifice 0.14.6+1.16.5
		autoconfig1u: Auto Config v1 Updated 3.3.1
		autofish: Autofish 0.9.2
		better-world-list: Better World List 1.0.4-1.16.2
		betterdroppeditems: Better Dropped Items 1.2.4-1.16.2
		blanket: Blanket 0.0.7
		blockmeter: Block Meter 1.5.4-beta
		blue_endless_jankson: jankson 1.2.0
		cleanview: CleanView 1.16.5-v1-fabric
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 1.5.47
		cloth-config2: Cloth Config v4 4.11.14
		cmdkeybind: Command Macros 1.5.2-1.16
		controlling: Controlling For Fabric 1.1.2
		cotton-client-commands: Cotton Client Commands 1.0.1+1.16-rc1
		crowdin-translate: CrowdinTranslate 1.2
		cullleaves: Cull Leaves 2.0.0
		dark-loading-screen: Dark Loading Screen 1.6.3
		de_javagl_obj: obj 0.3.0
		disable_custom_worlds_advice: Disable Custom Worlds Advice 1.2
		ears: Ears 1.2.2
		eggtab: Egg Tab 2.1.0+1.16-fabric
		enchantedtooltips: Enchanted ToolTips 1.3.4-1.16
		fabric: Fabric API 0.32.5+1.16
		fabric-api-base: Fabric API Base 0.2.1+9354966b7d
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.0.0+dc716ea17d
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.1+ca58154a7d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+ca58154a7d
		fabric-command-api-v1: Fabric Command API (v1) 1.1.1+351679a77d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.11+9354966b7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-dimensions-v1: fabric-dimensions-v1 2.0.6+9354966b7d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.0.3+ca58154a7d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.2+ca58154a7d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.5+351679a77d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+9354966b7d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+ca58154a7d
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-models-v0: Fabric Models (v0) 0.2.1+ca58154a7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.1+ca58154a7d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.4+9354966b7d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.1+ca58154a7d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.4+ca58154a7d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.1+ca58154a7d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+ca58154a7d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+ca58154a7d
		fabric-rendering-v1: Fabric Rendering (v1) 1.5.1+ca58154a7d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.2+ca58154a7d
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.0+c045166c7d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.5+9354966b7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.4+ca58154a7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.6+ca58154a7d
		fabrication: Fabrication 1.3.4
		fabricloader: Fabric Loader 0.11.2
		fiber: fiber 0.23.0-2
		figura: Figura 0.0.4n
		hatlist: Hat List 1.1-SNAPSHOT
		held-item-info: Held Item Info 1.1.3
		horsestatsvanilla: Horse Stats Vanilla 4.1.6
		hydrogen: Hydrogen 0.2
		iamverysmart: I Am Very Smart 2.1.0+mc1.16.1
		illuminations: Illuminations 1.4.9
		inventoryprofiles: Inventory Profiles 0.4.2
		itemmodelfix: Item Model Fix 1.0.0
		jankson: Jankson 3.0.1+j1.2.0
		java: OpenJDK 64-Bit Server VM 11
		kirin: Kirin UI 1.7-1.16.2
		kyrptconfig: Kytpt Config 1.1.5-1.16
		lambdacontrols: LambdaControls 1.5.0+1.16.2
		lambdynlights: LambDynamicLights 1.3.4+1.16
		lazydfu: LazyDFU 0.1.2
		libgui: LibGui 3.3.5+1.16.5
		lightoverlay: Light Overlay 5.8.0
		litematica: Litematica 0.0.0-dev.20210120.213004
		lithium: Lithium 0.6.4
		malilib: MaLiLib 0.10.0-dev.21+arne.5
		mambience: MAmbience 3.0.0+1.16.5
		minecraft: Minecraft 1.16.5
		minihud: MiniHUD 0.19.0-dev.20201103.184029
		mixintrace: MixinTrace 1.0.0
		mm: Manningham Mills 2.1
		modmenu: Mod Menu 1.16.9
		mousewheelie: Mouse Wheelie 1.6.4+mc1.16.4
		multiconnect: MultiConnect 1.3.34
		multiconnect-api: MultiConnect API 1.3.34
		okzoomer: Ok Zoomer 4.0.1+1.16.2
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_codehaus_groovy_groovy: groovy 3.0.3
		org_codehaus_groovy_groovy-jsr223: groovy-jsr223 3.0.3
		org_luaj_luaj-jse: luaj-jse 3.0.1
		org_yaml_snakeyaml: snakeyaml 1.26
		perspectivemod: Perspective Mod Redux 0.0.5
		phosphor: Phosphor 0.7.1+build.11
		presencefootsteps: Presence Footsteps r29+1.16.4
		pride: Pride Lib 1.0.0+1.16
		right-click-clear: Right Click Clear 2.0.2
		roughlyenoughitems: Roughly Enough Items 5.10.184
		roughlyenoughitems-api: REI (API) 5.10.184
		roughlyenoughitems-default-plugin: REI (Default Plugin) 5.10.184
		roughlyenoughitems-runtime: REI (Runtime) 5.10.184
		roughlyenoughresources: Roughly Enough Resources 2.1.3
		slight-gui-modifications: 'Slight' GUI Modifications 1.7.1
		smoothboot: Smooth Boot 1.16.4-1.4.0
		smoothscrollingeverywhere: Smooth Scrolling Everywhere 3.0.3-unstable
		spruceui: SpruceUI 2.0.4+1.16
		squake: Squake 2.0.0
		tillitbreaks: Till it Breaks 0.1
		torohealth: ToroHealth Damage Indicators 1.16.4-fabric-4
		transparent: Transparent 2.1.0
		tweakeroo: Tweakeroo 0.10.0-dev.20210209.145902
		tweed: Tweed API 3.0.0-beta.22
		vanguard: Vanguard 1.0.5
		viafabric: ViaFabric 0.3.1-SNAPSHOT+build-715.1-mc-1.16
		viaversion: ViaVersion 3.2.1
		wthit: wthit 2.3.0+1.16.5
	Launched Version: MultiMC5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Radeon RX Vega GL version 4.6.14760 Compatibility Profile Context 21.2.3 27.20.14535.3005, ATI Technologies Inc.
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, slightguimodifications:cts_textures, Fabric Mods
	Current Language: English (US)
	CPU: 16x AMD Ryzen 7 2700 Eight-Core Processor 
</pre>
</details>

--- a/platform-fabric-1.16/src/main/java/com/unascribed/ears/mixin/MixinPlayerSkinTexture.java
+++ b/platform-fabric-1.16/src/main/java/com/unascribed/ears/mixin/MixinPlayerSkinTexture.java
@@ -34,7 +34,9 @@ public abstract class MixinPlayerSkinTexture extends ResourceTexture implements 
 	private void loadTexture(InputStream stream, CallbackInfoReturnable<NativeImage> ci) {
 		EarsLog.debug("Platform:Inject", "Process player skin");
 		NativeImage cur = ci.getReturnValue();
-		earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		if (cur != null) {
+			earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		}
 	}
 	
 	private static boolean ears$reentering = false;

--- a/platform-fabric-1.17/src/main/java/com/unascribed/ears/mixin/MixinPlayerSkinTexture.java
+++ b/platform-fabric-1.17/src/main/java/com/unascribed/ears/mixin/MixinPlayerSkinTexture.java
@@ -34,7 +34,9 @@ public abstract class MixinPlayerSkinTexture extends ResourceTexture implements 
 	private void loadTexture(InputStream stream, CallbackInfoReturnable<NativeImage> ci) {
 		EarsLog.debug("Platform:Inject", "Process player skin");
 		NativeImage cur = ci.getReturnValue();
-		earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		if (cur != null) {
+			earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		}
 	}
 	
 	private static boolean ears$reentering = false;

--- a/platform-forge-1.15/src/main/java/com/unascribed/ears/mixin/MixinDownloadingTexture.java
+++ b/platform-forge-1.15/src/main/java/com/unascribed/ears/mixin/MixinDownloadingTexture.java
@@ -34,7 +34,9 @@ public abstract class MixinDownloadingTexture extends SimpleTexture implements E
 	private void loadTexture(InputStream stream, CallbackInfoReturnable<NativeImage> ci) {
 		EarsLog.debug("Platform:Inject", "Process player skin");
 		NativeImage cur = ci.getReturnValue();
-		earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		if (cur != null) {
+			earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		}
 	}
 	
 	private static boolean ears$reentering;

--- a/platform-forge-1.16/src/main/java/com/unascribed/ears/mixin/MixinDownloadingTexture.java
+++ b/platform-forge-1.16/src/main/java/com/unascribed/ears/mixin/MixinDownloadingTexture.java
@@ -34,7 +34,9 @@ public abstract class MixinDownloadingTexture extends SimpleTexture implements E
 	private void loadTexture(InputStream stream, CallbackInfoReturnable<NativeImage> ci) {
 		EarsLog.debug("Platform:Inject", "Process player skin");
 		NativeImage cur = ci.getReturnValue();
-		earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		if (cur != null) {
+			earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		}
 	}
 	
 	private static boolean ears$reentering;

--- a/platform-forge-1.17/src/main/java/com/unascribed/ears/mixin/MixinHttpTexture.java
+++ b/platform-forge-1.17/src/main/java/com/unascribed/ears/mixin/MixinHttpTexture.java
@@ -34,7 +34,9 @@ public abstract class MixinHttpTexture extends SimpleTexture implements EarsFeat
 	private void load(InputStream stream, CallbackInfoReturnable<NativeImage> ci) {
 		EarsLog.debug("Platform:Inject", "Process player skin");
 		NativeImage cur = ci.getReturnValue();
-		earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		if (cur != null) {
+			earsFeatures = EarsFeatures.detect(new RawEarsImage(cur.makePixelArray(), cur.getWidth(), cur.getHeight(), false), EarsStorage.get(cur, EarsStorage.Key.ALFALFA));
+		}
 	}
 	
 	private static boolean ears$reentering;

